discovery-provider/plugins/notifications/src/processNotifications/mappers/repostOfRepost.ts

```diff
@@ -120,10 +120,9 @@ export class RepostOfRepost extends BaseNotification<RepostOfRepostNotificationR
             )
           })
         )
-        // TODO: increment badge count
+        await this.incrementBadgeCount(this.receiverUserId)
```

Comment:
nice, is this something i need to consider for the trending notifs?

Reply:
yes! i thiiink this is like the count of notifications the user has waiting for them so all notifs should increment this number (but would confirm with @jowlee)